### PR TITLE
update README

### PR DESCRIPTION
**Pull request by hsnice16**
This PR will add the updated README in the development branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,24 +16,32 @@ Are you even a Marvel fan? Let's Check with marvelsQuiz
...
+- Home (<a href="https://marvelsquiz.netlify.app/" target="_blank">marvelsQuiz Home</a>)
+
 - Rules for each category
-  + Category One ([Avengers: Infinity War Rules](https://marvelsquiz.netlify.app/pages/avengers-infinity-war/rules.html))
-  + Category Two ([Doctor Strange Rules](https://marvelsquiz.netlify.app/pages/doctor-strange/rules.html))
-  + Category Three ([Captain Marvel Rules](https://marvelsquiz.netlify.app/pages/captain-marvel/rules.html))
+
+  - Category One (<a href="https://marvelsquiz.netlify.app/pages/avengers-infinity-war/rules.html" target="_blank">Avengers: Infinity War Rules</a>)
+  - Category Two (<a href="https://marvelsquiz.netlify.app/pages/doctor-strange/rules.html" target="_blank">Doctor Strange Rules</a>)
+  - Category Three (<a href="https://marvelsquiz.netlify.app/pages/captain-marvel/rules.html" target="_blank">Captain Marvel Rules</a>)
```
Comment (anik31) [inline README.md]: target="_blank" is not supported in github markdown, so you can remove it and instead use the markdown syntax for links.

Like this -
image

File: README.md [hunk 2]
```diff
@@ -16,24 +16,32 @@ Are you even a Marvel fan? Let's Check with marvelsQuiz
 
 ## Contains
```
Comment (anik31) [inline README.md:18]: suggestion - instead of contains you can write features

**Review by anik31 — COMMENTED** [review]
The readme, logo & badges look great 👍 

Suggestion - put the gif at the top just before the contains

**Review by sainath7878 — APPROVED** [review]
Readme looks good to me. Keep up the good work.

File: README.md (hunk 1 above)
Comment (hsnice16) [reply README.md]: Thanks for letting me know.

**Comment by hsnice16** [comment]
Thank you for reviewing this.

**Pull request merged**